auth-tokens/src/test/java/com/palantir/tokens/auth/UnverifiedJsonWebTokenTests.java

```diff
@@ -26,49 +26,50 @@
 
 final class UnverifiedJsonWebTokenTests {
 
-    private static final BearerToken SESSION_TOKEN = BearerToken.valueOf("header."
```

Comment:
These tests are currently designed based on our internal token types (session vs API). This is an implementation details that consumers should not be concerned with. I've refactored these tests to better reflect the guarantees and assumptions of this library by itself.